### PR TITLE
Survey/benchmark paper on different object localization methods

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: deep-learning/_posts/2017-09-07-objDetectorSurvey.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<div style="text-align:center">
+<img src="/deep-learning/images/objDetector17/sc03.png" width="500">
+</div>
+![](/deep-learning/images/objDetector17/sc04.png)
+
+
+## Conclusion
+
+- Fig.4a shows that **Faster-RCNN + inception ResNet v2** is the top performing configuration.
+- Fig.4b shows that an input resolution of 300 is best for large objects and 600 is best for small objects.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-07-objDetectorSurvey.md]: I disagree. In the plot you can see that green points are in the same region as the blue points on the y axis. Furthermore, the points that are the farthest on the y axis are green. I would conclude that 600 is always better, which seems intuitive. The main drawback must be computation speed.

File: deep-learning/_posts/2017-09-07-objDetectorSurvey.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- Fig.6 shows that using more than **50 box proposals** do not improve mAP while requiring more processing power. 
+- Table 4 shows that their top method acheives a mAP of .347 on COCO while ensemble methods reach 0.416, the best result ever.
```
[the remark this answers is not in the document]
Comment (lemairecarl) [reply deep-learning/_posts/2017-09-07-objDetectorSurvey.md]: 0.347 (consistency)

File: deep-learning/_posts/2017-09-07-objDetectorSurvey.md (hunk 1 above)
Comment (lemairecarl) [reply deep-learning/_posts/2017-09-07-objDetectorSurvey.md]: Ok I re-read your sentence and you're probably saying that 300 should only be used when we care about large objects. But at first I understood that 300 is the best resolution for detecting large objects (which is false).

File: deep-learning/_posts/2017-09-07-objDetectorSurvey.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<img src="/deep-learning/images/objDetector17/sc03.png" width="500">
+</div>
+![](/deep-learning/images/objDetector17/sc04.png)
+
+
+## Conclusion
+
+- Fig.4a shows that **Faster-RCNN + inception ResNet v2** is the top performing configuration.
+- Fig.4b shows that an input resolution of 300 is best for large objects and 600 is best for small objects.
+- Fig.6 shows that using more than **50 box proposals** do not improve mAP while requiring more processing power.
```
Comment (ppoulin91) [inline deep-learning/_posts/2017-09-07-objDetectorSurvey.md]: "does not improve"

File: deep-learning/_posts/2017-09-07-objDetectorSurvey.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+<div style="text-align:center">
+<img src="/deep-learning/images/objDetector17/sc03.png" width="500">
+</div>
+![](/deep-learning/images/objDetector17/sc04.png)
+
+
+## Conclusion
+
+- Fig.4a shows that **Faster-RCNN + inception ResNet v2** is the top performing configuration.
+- Fig.4b shows that an input resolution of 300 should only be use when we car about small objects.
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-09-07-objDetectorSurvey.md]: Does not make sense... In a smaller resolution small objects tend to disappear...

Also, typo: car -> care

**Pull request merged**